### ISSUE TITLE
Checked task won't dismiss

### ISSUE THREAD

**Issue by shawnajean** [HabitRPG/habitica#5689]
I've got three items that won't dismiss from my To-Do list. They also haven't turned dark red like the other items around them of the same age.

![screen shot 2015-07-28 at 10 46 18 pm](https://cloud.githubusercontent.com/assets/775884/8949560/e20d9fe8-357a-11e5-8bb9-17a5dad13452.png)

When I check them, I'm awarded the coins and any drops, but then the item reappears (sometimes not until after a reload.) I am also unable to delete them from the list.

There aren't any errors in the console.

Running Chrome 44.0.2403.89 and Firefox 39 on Yosemite . User ID - bd351656-e26a-4e18-a69c-08221ae59ada

**Comment by Alys** [comment]
Note to devs:
There are 17 To-Dos ID numbers that each belong to two To-Dos (i.e., duplicated To-Do IDs).

@shawnajean Can you remember how long this has been happening for?

To fix it, first go to the "Done" tab of you To-Do column and click on the "Delete Completed" button at the bottom. This will permanently delete To-Dos that you have completed, so if you want to keep a record of them, [export your data](http://habitica.wikia.com/wiki/Data_Export) first. You are welcome to ask about that here if you wish! 

That alone might fix the problem. If it does not, when you notice a To-Do that won't stay completed or that can't be deleted, look down the list until you find another To-Do with the same text. It's likely that you'll be able to delete the bottom-most To-Do in a pair of duplicates. After you've deleted it, the remaining To-Do should behave normally.

I'm happy to help further with this, although I won't have much time free for the next 24 hours. When you've clicked on the "Delete Completed" button, post here to say you've done that, and I'll look at your To-Do ID numbers again to see if all of the duplicates have been removed.

**Comment by shawnajean** [comment]
Thanks for the quick response @Alys! Deleting the completed To-Dos did let me complete the tasks, so it seems fixed on my end!

I'm not sure how long it's been happening, but I do remember several duplicate tasks in the list around the time I added these ones. It was a bit ago.. I thought that I had added the doubles, cuz I do that sometimes, so I'm sure I changed the text on a few of them to different things.

**Comment by Alys** [comment]
@shawnajean Thanks for letting us know!

If it happens again, we would REALLY like to hear about it, even if you're able to fix it yourself. You could comment again here or in a new issue, as you prefer. We think we fixed the problem that allowed this to happen, so I'm hoping that your duplicates were created before that fix, and that you won't see it again.
